api/client-server/key_backup.yaml

```
@@ -0,0 +1,761 @@
+            type: object
+            properties:
+              version:
+                type: string
+                description: The backup version
+                example: "1"
+            required:
+              - version
+      tags:
+        - End-to-end encryption
+  "/room_keys/version/{version}":
+    get:
+      summary: Get information about an existing backup.
+      description: |-
+        Get information about an existing backup.
+      operationId: getRoomKeysVersion
+      security:
+        - accessToken: []
+      parameters:
+        - in: path
+          type: string
+          name: version
+          description: |-
+            The backup version to get. When an empty string, the trailing slash
+            is optional, and the current backup is returned.
+          required: true
+          x-example: "1"
+      responses:
+        200:
+          description:
+            The information about the requested backup.
+          schema:
+            type: object
+            properties:
+              algorithm:
+                type: string
+                description: The algorithm used for storing backups.
+                enum: ["m.megolm_backup.v1.curve25519-aes-sha2"]
+                example: "m.megolm_backup.v1.curve25519-aes-sha2"
+              auth_data:
+                description: |-
+                  Algorithm-dependent data. See the documentation for the backup
+                  algorithms in `Server-side key backups`_ for more information on the
+                  expected format of the data.
+                type: object
+                example: {
+                    "public_key": "abcdefg",
+                    "signatures": {
+                      "@alice:example.org": {
+                        "ed25519:deviceid": "signature"
+                      }
+                    }
+                  }
+              count:
+                description: The number of keys stored in the backup.
+                type: integer
+                example: 42
+              etag:
+                description: |-
+                  An opaque string representing stored keys in the backup.
+                  Clients can compare it with the ``etag`` value they received
+                  in the request of their last key storage request.  If not
+                  equal, another client has modified the backup.
+                type: string
+                example: "anopaquestring"
+              version:
+                type: string
+                description: The backup version
+                example: "1"
+            required:
+              - algorithm
+              - auth_data
+              - count
+              - etag
+              - version
+        404:
+          description:
+            The backup specified does not exist
+          examples:
+            application/json: {
+                "errcode": "M_NOT_FOUND",
+                "error": "Unknown backup version"
+              }
```

The indentation this was probably copied from is an autoconvert fail from years ago
```suggestion
              "errcode": "M_NOT_FOUND",
              "error": "Unknown backup version"
            }
```